oauth2server/apps/accounts/decorators.py

```diff
@@ -64,17 +70,17 @@ def _extract_client(request):
 
         # First, let's check Authorization header if present
         if 'HTTP_AUTHORIZATION' in request.META:
-            print "*** We have a HTTP_AUTHORIZATION request ***. Which is: {}".format(request.META['HTTP_AUTHORIZATION'])
+            stdlogger.debug("We have a HTTP_AUTHORIZATION request ***. Which is: {}".format(request.META['HTTP_AUTHORIZATION']))
             auth_header = request.META['HTTP_AUTHORIZATION']
             auth_method, auth = re.split(':|;|,| ', auth_header)
             #auth_method, auth = request.META['HTTP_AUTHORIZATION'].split(':')
             if auth_method.lower() == 'basic':
                 client_id, client_secret = base64.b64decode(auth).split(':')
-                print "client id is: {} , and client secret is: {}".format(client_id, client_secret)
+                stdlogger.debug("client id is: {} , and client secret is: {}".format(client_id, client_secret))
```

Comment:
The secret and id should not be written out in case logging level is increased in prod.